src/style.css

```diff
@@ -0,0 +1,132 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  display: flex;
+}
+
+.card__info {
+  margin: 40px 16px 0;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  padding-left: 19px;
+  padding-top: 32px;
+}
+
+.card__info-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  font-style: normal;
+}
+
+.card__info-code {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+  left: 16px;
+}
+
+.card__info-review-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  width: 166px;
+  height: 16px;
+  margin-top: 16px;
+}
+
+.card__info-review-count {
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  font-weight: normal;
+  display: flex;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__info-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  width: 166px;
+  height: 18px;
+  margin-top: 24px;
+}
+
+.card__info-price-letter {
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__info-price-number {
+  font-size: 16px;
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 700;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+}
+
+.card__buy {
+  box-sizing: border-box;
+  display: block;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  color: #fff;
+  text-decoration: none;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  margin: 16px;
+  left: 16px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.card__buy:hover {
+  background-color: rgb(255, 255, 255);
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover

Reply:
done